src/server/api/workable.js

```diff
@@ -11,13 +11,12 @@ export default class Workable {
         'Content-Type': 'application/json'
       }
     }).then((response) => {
-      const filtered = response.jobs.map((job) => {
+      return response.body.jobs.map((job) => {
```

Comment:
This should be handled by our fetch proxy. However it does bring up an interesting point that this is not immediately obvious. Perhaps we could rework the fetch proxy into some helper functions to use in the promise chain? E.g.

``` javascript
return fetch('xyz')
  .then(handleHttpError)
  .then(myHandler)
  .catch(genericFetchErrorHandler) // if jobs does not exist this will map it to a 500
```